README.troubleshooting.inference.md

```diff
@@ -0,0 +1,53 @@
+
+## If you see a redis cleanup pod hanging there in the cluster for your deployment... 
+
+it means some redis error has happened during killing your pod. In this case, do:
+
+    kubectl get raycluster -n <xxx> to get the raycluster name for your deployment.
+
+    kubectl edit rayclusters.ray.io -n <xxx> <ray_cluster_name> to bring up the yaml for the deployment.
+
+    Find finalizers: - ray.io/gcs-ft-redis-cleanup-finalizer and delete those 2 lines and save.
```

Comment:
As these steps will only work _after_ a person is able to access the Kubernetes cluster via bastion host, I believe we should just link the [documentation](https://mozilla-hub.atlassian.net/wiki/spaces/DATA/pages/785514640/Deploy+inference+servers+to+production+GKE+using+Ray+Serve#Running-kubectl%2Fhelm-commands) for doing that here.